pwndbg/regs.py

```diff
@@ -276,6 +288,8 @@ def __getattr__(self, attr):
                 value = gdb77_get_register(attr)
                 value = value.cast(pwndbg.typeinfo.uint32)
             else:
+                if attr.lower() == 'xpsr':
+                    attr = 'xPSR'
```

Comment:
This special case was necessary for me to make this work on GDB8.3, which seems to be case-sensitive for this register.